CMakeLists.txt

```diff
@@ -391,7 +391,7 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   if(NOT SYSTEMCONFIGURATION_FRAMEWORK)
      message(FATAL_ERROR "SystemConfiguration framework not found")
   endif()
-  list(APPEND CURL_LIBS "${SYSTEMCONFIGURATION_FRAMEWORK}")
```

Comment:
Should we omit checking for framework presence and just expect it to be there?

Reply:
@gjasny why? I mean if the test is already there and working, does it cause any harm?

Reply:
It does not do any harm.